### DISCUSSION TITLE
It should be possible to specify the header containing the Client's IP for IP based functions

### DISCUSSION THREAD

**Discussion by ericharth-acer** [ExpressionEngine/ExpressionEngine#3573] [Feature Requests]
### Suggest an idea for ExpressionEngine

When EE is deployed behind a proxy or load balancer, the IP used by the system for IP based functions such as control panel login attempts only sees the load balancer's IP. This causes false positives when counting the number of logins per minute and incorrectly limits good/genuine login attempts.

Ideally, there would be a simple system wide config option to indicate which header contains the real Client IP value. Any IP based functions should use this header if specified. If the config option isn't specified the system could default to 'ClientIP'

**Comment by intoeetive** [comment]
The issue migh be resolved with this PR: https://github.com/ExpressionEngine/ExpressionEngine/pull/3470
